src/main/java/net/mcreator/ui/minecraft/modellayers/JModelLayerList.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2023, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.ui.minecraft.modellayers;
+
+import net.mcreator.element.types.LivingEntity;
+import net.mcreator.ui.MCreator;
+import net.mcreator.ui.component.util.PanelUtils;
+import net.mcreator.ui.help.IHelpContext;
+import net.mcreator.ui.init.L10N;
+import net.mcreator.ui.minecraft.JEntriesList;
+import net.mcreator.ui.validation.AggregatedValidationResult;
+
+import javax.swing.*;
+import java.awt.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class JModelLayerList extends JEntriesList {
+
+	private final List<JModelLayerListEntry> entryList = new ArrayList<>();
+
+	private final JPanel entries = new JPanel(new GridLayout(0, 1, 5, 5));
+
+	public JModelLayerList(MCreator mcreator, IHelpContext gui) {
+		super(mcreator, new BorderLayout(), gui);
+		setOpaque(false);
+
+		JPanel topbar = new JPanel(new FlowLayout(FlowLayout.LEFT));
+		topbar.setBackground((Color) UIManager.get("MCreatorLAF.LIGHT_ACCENT"));
+
+		add.setText(L10N.t("elementgui.living_entity.add_model_layer"));
+		topbar.add(add);
+
+		add("North", topbar);
+
+		entries.setOpaque(false);
+
+		add.addActionListener(e -> {
+			JModelLayerListEntry entry = new JModelLayerListEntry(mcreator, gui, entries, entryList);
+			registerEntryUI(entry);
+		});
+
+		add("Center", PanelUtils.pullElementUp(entries));

```

Comment:
Those are a bit nonstandard, including the fact there is no scroll pane so not sure what happens if one adds many layers

Reply:
I copy pasted this part from JPotionList so I didn't think it would be an issue.

Reply:
Hmm, interesting, then this is a bug. I will take care of others but this PR should also implement one for this case (after I fix others).

Thank you for clarifying this Nerdy!

Reply:
#4244 was merged, so this PR can now adapt the simple entries list

Reply:
Here you go.